### PR TITLE
Refactor rules

### PR DESCRIPTION
**Pull request by doctrino**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .PHONY: run-explorer run-tests run-linters build-ui build-python build-docker run-docker compose-up
 
-version="0.59.0"
```
Comment (doctrino) [inline Makefile:3]: Does this need a bump as we are not releasing this feature yet? Just adding the code.

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ Changes are grouped as follows:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+
+## [0.60.0] - 26-01-24
+
+## Added
+
+- Introduced `Rules` profiling which follow data modeling lifecycle
```
Comment (doctrino) [inline docs/CHANGELOG.md:23]: We are not officially introducing this feature yet. 
```suggestion
```

File: Makefile (hunk 1 above)
Comment (nikokaoja) [reply Makefile:3]: Agree !

File: docs/CHANGELOG.md (hunk 2 above)
Comment (nikokaoja) [reply docs/CHANGELOG.md]: done !

**Review by nikokaoja — APPROVED** [review]

**Pull request merged**